packages/manager/apps/cloud/client/components/at-internet/at-internet.config.js

```diff
@@ -1,32 +1,17 @@
+import { TRACKING } from './at-internet.constants';
+
 angular
   .module('managerApp')
-  .config((atInternetProvider, atInternetUiRouterPluginProvider, CONFIG) => {
-    const trackingEnabled = CONFIG.env === 'production';
-
-    atInternetProvider.setEnabled(trackingEnabled);
-    atInternetProvider.setDebug(!trackingEnabled);
-
-    atInternetUiRouterPluginProvider.setTrackStateChange(true);
-    atInternetUiRouterPluginProvider.addStateNameFilter((routeName) => {
-      const prefix = 'cloud';
-      const route = routeName ? routeName.replace(/\./g, '::') : '';
-      return `${prefix}::${route}`;
-    });
+  .config((atInternetConfigurationProvider) => {
```

Comment:
We can take the opportunity to add some missing ngInject I think.

```suggestion
  .config(/* @ngInject */ (atInternetConfigurationProvider) => {
```